jumpbox.sh

```diff
@@ -7,5 +7,8 @@ cat <<EOF >> $HOME/.bashrc
 PS1="\[\$(tput setaf "$PROMPT_COLOR")\]\[\e]0;\u@$BOSH_DIRECTOR_NAME: \w\a\]${debian_chroot:+($debian_chroot)}\u@$BOSH_DIRECTOR_NAME:\w\$ \[\$(tput sgr0)\]"
 EOF
 
+export CREDHUB_CA_CERT="$(pwd)/${CREDHUB_CA_CERT}"
```

Comment:
we pass relative paths as parameters because the base path varies from run to run. This converts the relative path to an absolute path, which should allow us to run bosh commands from any directory without issue.